src/server/scripts/Spells/spell_warrior.cpp

```diff
@@ -422,6 +424,46 @@ class spell_warr_frenzied_enrage : public SpellScript
     }
 };
 
+// 215568 - Fresh Meat (attached to 23881 - Bloodthirst)
+// 215568 - Fresh Meat (attached to 335096 - Bloodbath)
+class spell_warr_fresh_meat : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_WARRIOR_FRESH_MEAT_DEBUFF, SPELL_WARRIOR_FRESH_MEAT_TALENT, SPELL_WARRIOR_ENRAGE });
+    }
+
+    bool Load() override
+    {
+        return GetCaster()->HasAura(SPELL_WARRIOR_FRESH_MEAT_TALENT);
+    }
+
+    void ApplyFreshMeat(SpellEffIndex /*effIndex*/) const
+    {
+        Unit* target = GetHitUnit();
+        if (target != GetExplTargetUnit())
+            return;
+
+        Unit* caster = GetCaster();
+
+        if (target->HasAura(SPELL_WARRIOR_FRESH_MEAT_DEBUFF, caster->GetGUID()))
+            return;
+
+        caster->CastSpell(target, SPELL_WARRIOR_FRESH_MEAT_DEBUFF, CastSpellExtraArgsInit{
+            .TriggerFlags = TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR
+        });
+
+        caster->CastSpell(nullptr, SPELL_WARRIOR_ENRAGE, CastSpellExtraArgsInit{
+            .TriggerFlags = TRIGGERED_IGNORE_CAST_IN_PROGRESS | TRIGGERED_DONT_REPORT_CAST_ERROR
```

Comment:
also, spell 215568 applies SPELL_AURA_ADD_FLAT_MODIFIER at index 12 for 184361 - Enrage spell of value 15, it's probably the % increase chance for Bloodthirsty spell (this index probably is some type of identifier of spells that can trigger it, and Bloodthirsty is at 12).

Reply:
This modifier is already handled in https://github.com/TrinityCore/TrinityCore/commit/35214519ce12064223fb6c39fbdc2519db4569d3